part-two/Population.md

```diff
@@ -1,24 +1,22 @@
+Well Malthus was right about one thing: global population did explode. Population growth started to accelerate right at the time of his writing around 1800. Since then, the human population has grown from about 1B to over 7B people here on planet Earth {{"WORLDO" | cite}}. As an optimist, the thing to note immediately, though, is that Malthus&apos;s most dire fears about the implications of this population growth have not been realized. There has been no global scale starvation and even the fear that most people would live in abject poverty has not come true. In fact, the opposite has happened recently. Around the world the number of people living in extreme poverty has been declining all the while population growth has been about twice as fast as what Malthus predicted as an upper limit of 1 billion people added in 25 years {{"OWD3" | cite}}.
 
 What Malthus got wrong was the rate of technological progress. First, Malthus was wrong in being pessimistic about our ability to improve agricultural productivity. Here is just some of the amazing progress in agriculture since his writing. The percentage of the global workforce employed in agriculture has declined from more than 80% to 33% and is falling rapidly (in the US and other advanced economies agriculture represents 2% or less of employment). Globally in the last 50 years alone, the land required to produce the same output of food has declined by a stunning 68% {{"OWD1" | cite}}.
 
-Second, Malthus could not foresee the scientific breakthroughs that made the industrial revolution possible. That revolution not only powered the agricultural productivity increase but also gave us dramatic advances in the standard of living including much increased life expectancy, faster transportation, cheaper and better communication and so on.
+Second, Malthus could not foresee the scientific breakthroughs that made the industrial revolution possible. That revolution not only powered the agricultural productivity increase but also gave us dramatic advances in the standard of living, including much increased life expectancy, faster transportation, cheaper and better communication and so on.
 
-Just because Malthus was wrong so far isn&apos;t by itself a guarantee that his predictions couldn&apos;t catch up with us though. If population growth were to outstrip technological progress this would in fact be the case. We know this because we have seen it happen in India {{ "WIKI" | cite}} and other places that have experienced population growth in excess of progress resulting in mass starvation.
+Although just because Malthus was wrong so far isn&apos;t by itself a guarantee that his predictions couldn&apos;t catch up with us. If population growth were to outstrip technological progress this would in fact be the case. We know this because we have seen it happen in India {{ "WIKI" | cite}} and other places that have experienced population growth in excess of progress resulting in mass starvation.
```

Comment:
Don't think the extra "Although" helps. But you are right that my sentence isn't great. How about "Malthus being wrong so far ..."